README.rst

```diff
@@ -1,3 +1,7 @@
+.. image:: images/logo-reana.png
```

Comment:
The logo seems huge to me. WDYT about setting some [width or scale](https://docutils.sourceforge.io/docs/ref/rst/directives.html#image)?

Reply:
You are right, I've tried several combinations and nothing seems to be applied (I've used [the GitHub web editor on `README.rst`](https://github.com/diegodelemos/reana/edit/335-restructuring-docs/README.rst)  to test the preview). I guess we have to resize the image manually?

Reply:
Mmh weird, I've tried and I can reproduce what you say... perhaps we can embed html `<img>` somehow?

Reply:
We have impactful software, so big logo :wink: Anyway, check e.g. Luigi logo which is also pretty big, I think it is OK for MVP if scaling directives are being ignored...